packages/react-native/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Children, cloneElement, useMemo, useState } from 'react';
+import { View } from 'react-native';
+
+import { childIsValidComponent } from '../../utils';
+import { Button } from '../Button';
+
+import { styles } from './styles';
+import { TabProps, TabsProps } from './types';
+
+export function Tab({ title, ...rest }: TabProps): JSX.Element {
+  return (
+    <Button {...rest} accessibilityRole="tab">
+      {title}
+    </Button>
+  );
+}
+
+export function Tabs({
+  children,
+  defaultIndex,
+  onChange,
+  selectedIndex = 0,
```

Comment:
Why do we need both `defaultIndex` and `selectedIndex`?

Reply:
`defaultIndex` controls which Tab's content is initially displayed when using Tabs as an uncontrolled component. 

`selectedIndex` is the controlled index of the currently selected Tab. This should be used in tandem with `onChange` to use Tabs as a controlled component.

Reply:
Changed `Tabs` to only be a controlled component (for now)